prow/recreate_prow_configmaps.py

```diff
@@ -26,19 +26,20 @@
 
 from __future__ import print_function
 
-from argparse import ArgumentParser
 import os
 import sys
+
 import subprocess
+from argparse import ArgumentParser
```

Comment:
Please import modules (argparse) not methods/classes (ArgumentParser)

http://google.github.io/styleguide/pyguide.html#22-imports